pkg/northbound/gnmi/set.go

```diff
@@ -73,12 +73,16 @@ func (s *Server) Set(ctx context.Context, req *gnmi.SetRequest) (*gnmi.SetRespon
 		return nil, err
 	}
 	updateResults := make([]*gnmi.UpdateResult, 0)
-	configName := "Running"
 	for target, updates := range targetUpdates {
-		changeID, err := manager.GetManager().SetNetworkConfig(target, configName, updates, targetRemoves[target])
+		changeID, err := manager.GetManager().SetNetworkConfig(target, target, updates, targetRemoves[target])
```

Comment:
why is this changed to double target ?

Reply:
The thing is when you have more than 1 target each one should have its own Configuration - not just a shared "Running" one between the 2. Therefore I have given the configuration the same name as the target - maybe should change it to "<targetName>Running". If you want to see the effect just change it back to what it was and see the Unit test fail.